### DISCUSSION TITLE
更新日志

### DISCUSSION THREAD

**Discussion by ghost** [hexo-theme-Acrylic/Hexo-Theme-Acrylic#19] [Announcements]
以后将在此提出出现的bug，已经修复的bug

**Comment by ShineYull** [comment]
后端：vercel已知普通用户不支持超过10秒的请求，待修复。

**Reply by ShineYull** [reply]
vercel官方说明文档：https://vercel.com/docs/concepts/limits/overview

**Reply by ShineYull** [reply]
也已知vercel不支持websocket
参考：https://github.com/vercel/community/discussions/422

**Comment by ghost** [comment]
Acrylic-Promote 开始开发优化，版本号归零
